subprojects/packagefiles/sdl3_ttf/meson.build

```diff
@@ -0,0 +1,38 @@
+project(
+  'sdl3_ttf',
+  'c',
+  license: 'zlib',
+  version: '3.2.2',
+)
```

Comment:
Either needs `meson_version: '>=0.50.0'` or `include_directories:` arguments need to be wrapped in `include_directories()`.